packages/cli/src/build/rollup.ts

```diff
@@ -31,7 +31,12 @@ const makeExternalPredicate = (externalArr: string[]) => {
   return (id: string) => pattern.test(id);
 };
 
-export type RollupConfigType = "umd" | "browser" | "node-dev" | "node-prod";
+export type RollupConfigType =
+  | "umd"
+  | "browser"
+  | "node-dev"
+  | "node-prod"
+  | "worker";
```

Comment:
This will come at the cost of some duplication but I believe that if we are going to use the `package.json#exports` field now (as opt-in, most likely) then we should output dev/prod bundles separately for all supported environments/conditions. The consumer of those shouldn't have to care about replacing stuff like `process.env.NODE_ENV` to use those - they should just set the conditions in their bundler of choice and that's it.

To speed the bundling stuff we could consider using Rollup's output plugins - but those are applied **after** tree-shaking and all is applied. So even though this could result in some perf gains it could potentially lead to suboptimal bundles. Since we are mostly recommending to only build on CI the perf concern might not be that important.

Reply:
I've added support for "production" and "development" builds for each environment.